.github/CODEOWNERS

```
@@ -743,6 +743,14 @@ i18next.config.ts @grafana/grafana-frontend-platform
 
 # @grafana/api-clients
 /packages/grafana-api-clients/ @grafana/grafana-frontend-platform @grafana/grafana-search-navigate-organise
+/packages/grafana-api-clients/src/clients/rtkq/advisor/ @grafana/plugins-platform-frontend
+/packages/grafana-api-clients/src/clients/rtkq/correlations/ @grafana/datapro
+/packages/grafana-api-clients/src/clients/rtkq/dashboard/ @grafana/dashboards-squad
+/packages/grafana-api-clients/src/clients/rtkq/folder/ @grafana/grafana-search-and-storage @grafana/grafana-search-navigate-organise
+/packages/grafana-api-clients/src/clients/rtkq/iam/ @grafana/access-squad
```

Could you also add the identity squad here please, they own some of the APIs.
```suggestion
/packages/grafana-api-clients/src/clients/rtkq/iam/ @grafana/access-squad @grafana/identity-squad
```

---

🙏 Thanks!